Core/examples/Main.cpp

```diff
@@ -108,6 +114,9 @@ void buildSceneGraph(GLState& glState)
 
     auto tunnelScene = Buffer::construct(1024, 768, "tunnelScene", vertexShader, fragmentTunnelScene);
 
+    auto baseScene = Buffer::construct(1024, 768, "baseScene", vertexShader, fragmentBaseScene);
+
+    glState.addScene(Scene{ baseScene, timer::ms_t(3000000) });
```

Comment:
maybe it would be nicer with timer::milliseconds(300000) instead of the cryptic ms_t. (not for this PR but later)
This particular could be in seconds though..